### PR TITLE
feat(PowerInfoList): Add PowerInfoList-related components

### PR DESCRIPTION
**Pull request by grandq33769**

### CODE REVIEW

**Comment by ttw225** [comment]
@wiasliaw77210 page updated

File: components/power_info/PowerInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        title={'電力資訊'}
+        options={{
+          search: false,
+          headerStyle: {
+            textAlign: 'center',
+          },
+        }}
+        tableRef={tableRef}
+        columns={state.columns}
+        data={query =>
```
Comment (wiasliaw77210) [inline components/power_info/PowerInfo.tsx]: Please simplify this query api.

File: typing/PowerInfoType.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export enum PowerType {
```
[the remark this answers is not in the document]
Comment (grandq33769) [reply typing/PowerInfoType.ts]: Fix in 2c13b2c .

File: components/power_info/PowerInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+import styled from 'styled-components';
+
+const PowerInfoHeaders = {
+  'Content-Type': 'application/json',
+  Authorization: 'Bearer xxx',
+};
+
+const url_powerinfo = 'http://140.116.247.120:5000/power_info';
```
[the remark this answers is not in the document]
Comment (grandq33769) [reply components/power_info/PowerInfo.tsx]: Fix in a6298e1 .

File: components/power_info/PowerInfo.tsx (hunk 1 above)
Comment (ttw225) [reply components/power_info/PowerInfo.tsx]: Need some help or suggestions.

Comment (ttw225) [reply components/power_info/PowerInfo.tsx]: Fix in df610ad.

**Pull request closed without merging**